openedx/core/djangoapps/user_api/accounts/settings_views.py

```diff
@@ -148,7 +148,7 @@ def account_settings_context(request):
     }
 
     enterprise_customer = get_enterprise_customer_for_learner(user=request.user)
-    update_account_settings_context_for_enterprise(context, enterprise_customer)
+    update_account_settings_context_for_enterprise(context, enterprise_customer, user)
```

Comment:
nit: little code improve could be moving this statment
```
enterprise_customer = get_enterprise_customer_for_learner(user=request.user)
```
inside `update_account_settings_context_for_enterprise` method since `enterprise_customer ` is being used in that method only